packages/rancher-monitoring/rancher-monitoring/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/prometheus-community/helm-charts.git
 subdirectory: charts/kube-prometheus-stack
 commit: beaa76a4ac26584c6cc77c27bc0d40769fb98b26
-version: 104.0.2
```

Comment:
Note: This version isn't final, it's been added for testing and can be changed based off of any other monitoring work I may not be aware of